include/libdnf5/base/goal_elements.hpp

```diff
@@ -137,12 +140,22 @@ enum class GoalAction {
     REASON_CHANGE,
     ENABLE,
     DISABLE,
-    RESET
+    RESET,
+    REPLAY_INSTALL,
+    REPLAY_REMOVE,
+    REPLAY_UPGRADE,
+    REPLAY_REINSTALL,
+    REPLAY_REASON_CHANGE,
+    REPLAY_REASON_OVERRIDE,
+    REVERT_COMPS_UPGRADE
 };
 
 /// Convert GoalAction enum to user-readable string
 std::string goal_action_to_string(GoalAction action);
 
+/// Check whether the action is a replay action
+bool goal_action_is_replay(GoalAction action);
```

Comment:
Are you sure that you want to have this method in API? It is very specific and it is only used in `dnf5/main.cpp`.

Reply:
I am so sorry, I've overlooked multiple usage somewhere else. Please ignore the comment above.